packages/targeting/src/targeting.ts

```diff
@@ -0,0 +1,21 @@
+import { EvaluationEngine } from '@amplitude/experiment-core';
+import { Targeting as AmplitudeTargeting, TargetingParameters } from './typings/targeting';
+
+export class Targeting implements AmplitudeTargeting {
+  evaluationEngine: EvaluationEngine;
+
+  constructor() {
+    this.evaluationEngine = new EvaluationEngine();
+  }
+
+  evaluateTargeting({ event, userProperties, deviceId, flag }: TargetingParameters) {
+    const context = {
+      event,
+      user: {
+        device_id: deviceId,
+        user_properties: userProperties,
+      },
+    };
```

Comment:
In your test you're bucketing by session id, but that is not included in this context. Most likely you want to add that here.

Reply:
Ah thanks for pointing that out, will update